assets/scripts/build_gallery.py

```diff
@@ -1,11 +1,95 @@
+import ast
 import importlib
 import json
 import logging
 import os
 import sys
+import traceback
+from subprocess import CalledProcessError, CompletedProcess, run
 
-logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
+chandler = logging.StreamHandler(stream=sys.stdout)
+chandler.setLevel(logging.DEBUG)
+chandler.setFormatter(
+    logging.Formatter("%(asctime)s - %(levelname)s - %(message)s", "%Y-%m-%dT%H:%M:%S")
+)
+logger.addHandler(chandler)
+logger.setLevel(logging.DEBUG)
+
+
+def execute_shell_command(command: str) -> int:
```

Comment:
How is this different from subprocess.run()? (If there's some relevant different, I'd add an explanation to the docstring)

Reply:
Added a note in the docstring -- it simply provides Exception handling